openhtf/io/test_record.py

```diff
@@ -26,38 +29,19 @@ class InvalidMeasurementDimensions(Exception):
   """Raised when a measurement is taken with the wrong number of dimensions."""
 
 
-class TestRecord(collections.namedtuple(
-    'TestRecord', 'dut_id station_id '
-                  'start_time_millis end_time_millis '
-                  'outcome outcome_details '
-                  'metadata openhtf_version code phases log_records')):
-  """The record of a single run of a test."""
-  def __new__(cls, test_filename, docstring, test_code, dut_id, station_id):
-    # TODO(jethier): Fill in openhtf_version
-    self = super(TestRecord, cls).__new__(
-        cls,
-        dut_id, station_id,        # dut_id, station_id
-        util.TimeMillis(), None,  # start/end
-        None, [],                  # outcome and details
-        {}, None, test_code,       # metadata, version, code
-        [], [])                    # phases and log_records
-    self.metadata['filename'] = test_filename
-    self.metadata['docstring'] = docstring
-    return self
-
-  @classmethod
-  def FromFrame(cls, frame):
-    """Create a TestRecord, initialized from the given stack frame.
+TestMetadata = mutablerecords.Record(
```

Comment:
This seemed pretty trivial, and can easily be undone in this PR by replacing 'TestMetadata' below with 'dict'